### PR TITLE
Security UI issue - More tab is displayed incorrectly

### PR DESCRIPTION
**Pull request by mikebigun**
### Steps to reproduce

1. Login as a SuperUser 
2. Navigate to Persona bar -> Mouse hover on Settings 
3. Click on Security

Actual result:

image

Expected:

image

### CODE REVIEW

File: Extensions/Settings/Dnn.PersonaBar.Security/Security.Web/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "@dnnsoftware/dnn-react-common": "^2.0.56",
     "babel-loader": "8.0.4",
     "babel-plugin-transform-react-remove-prop-types": "0.4.20",
+    "create-react-class": "^15.6.3",
```
Comment (valadas) [inline Extensions/Settings/Dnn.PersonaBar.Security/Security.Web/package.json:19]: Is this required, was there a build warning or something ?

Comment (mikebigun) [reply Extensions/Settings/Dnn.PersonaBar.Security/Security.Web/package.json:19]: Yeap, there is a warning:

image

**Review by daguiler — DISMISSED** [review]

**Comment by mikebigun** [comment]
Just added yarn.lock updated with `create-react-class` dependency.
@daguiler, sorry, need your approval again.

File: Extensions/Settings/Dnn.PersonaBar.Security/Security.Web/yarn.lock [hunk 2]
```diff
@@ -1509,6 +1509,15 @@ create-hmac@^1.1.0, create-hmac@^1.1.2, create-hmac@^1.1.4:
     safe-buffer "^5.0.1"
     sha.js "^2.4.8"
 
+create-react-class@^15.6.3:
+  version "15.6.3"
+  resolved "https://www.myget.org/F/dnn-software-public/npm/create-react-class/-/create-react-class-15.6.3.tgz#2d73237fb3f970ae6ebe011a9e66f46dbca80036"
```
Comment (daguiler) [inline Extensions/Settings/Dnn.PersonaBar.Security/Security.Web/yarn.lock]: Can you change to `https://registry.yarnpkg.com` like in the rest of the cases?

Comment (mikebigun) [reply Extensions/Settings/Dnn.PersonaBar.Security/Security.Web/yarn.lock]: @daguiler, fixed.

**Review by valadas — APPROVED** [review]

**Review by daguiler — APPROVED** [review]

**Pull request merged**